packages/inventory/src/containers/ProfileView/index.tsx

```diff
@@ -0,0 +1,253 @@
+      <p className="item">
+        {label}: {value}
+      </p>
+    </Col>
+  );
+};
+
+/** component that renders service point list
+ *
+ * @param props - the component props
+ */
+const ServicePointProfile = (props: ServicePointsProfileTypes) => {
+  const { columns, opensrpBaseURL, service } = props;
+  const { broken, errorMessage, handleBrokenPage } = useHandleBrokenPage();
+  const [isLoading, setIsLoading] = useState<boolean>(true);
+  const dispatch = useDispatch();
+  const params = useParams<{ [INVENTORY_SERVICE_POINT_PROFILE_PARAM]: string }>();
+  const spId = params[INVENTORY_SERVICE_POINT_PROFILE_PARAM];
+
+  const inventoriesArray = useSelector((state) =>
+    getInventoriesByExpiry(state, { expired: false })
+  ) as Inventory[];
+  const structures = useSelector((state) =>
+    structuresSelector(state, {
+      searchQuery: undefined,
+      isJurisdiction: false,
+    })
+  );
+  const [structure] = structures.filter((loc) => loc.id === spId);
+  const trees = useSelector((state) => treesSelector(state, {}));
+
+  useEffect(() => {
+    // get structures, this is the most important call for this page
+    const params = {
+      serverVersion: 0,
+      // eslint-disable-next-line @typescript-eslint/camelcase
+      is_jurisdiction: false,
+    };
+    const structuresDispatcher = (locations: LocationUnit[] = []) => {
+      return dispatch(fetchLocationUnits(locations, false));
+    };
+    loadJurisdictions(
+      structuresDispatcher,
+      opensrpBaseURL,
+      params,
+      {},
+      service,
+      LOCATIONS_GET_ALL_SYNC_ENDPOINT
+    ).catch((err: Error) => {
+      handleBrokenPage(err);
+      setIsLoading(false);
+    });
+    // get root Jurisdictions so we can later get the trees.
+    const jurisdictionsDispatcher = (locations: LocationUnit[] = []) => {
+      return dispatch(fetchLocationUnits(locations, true));
+    };
+    loadJurisdictions(
+      jurisdictionsDispatcher,
+      opensrpBaseURL,
+      undefined,
+      undefined,
+      service
+    ).catch((err: Error) => sendErrorNotification(err.message));
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  useEffect(() => {
+    const serve = new OpenSRPService(`${GET_INVENTORY_BY_SERVICE_POINT}${spId}`, opensrpBaseURL);
+    serve
+      .list()
+      .then((res: Inventory[]) => {
+        dispatch(fetchInventories(res));
+        setIsLoading(false);
+      })
+      .catch(() => sendErrorNotification(ERROR_OCCURRED));
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [opensrpBaseURL, spId]);
+
+  if (isLoading) return <Spin size="large" />;
+
+  if (broken) {
+    return <BrokenPage errorMessage={errorMessage} />;
+  }
+
+  if (!(structure as LocationUnit | undefined) && spId) {
+    return <Resource404 />;
+  }
+
+  const pageTitle = `${SERVICE_POINT_INVENTORY}`;
+  const nodePath = getNodePath(structure, trees);
+
+  return (
+    <>
+      <div className="inventory-profile-header">
```

Comment:
Looking at the wireframe, I think it's best if this profile header was a component on its own, separate from the list of inventories. Also, could we name it more specific than `ProfileView` e.g `ServicePointProfile` cc @p-netm. I would prefer if this component had two children components one responsible for displaying the header, the other for the list of inventory items

Reply:
On second thought, I think its best if it was a `ServicePointProfile` component with an `InventoryList` component

Reply:
Created a separate component for Inventory table, named as `InventoryList`